src/main/archetype/it.launcher/pom.xml

```diff
@@ -223,15 +223,15 @@
                                 <start.bundles.timeout.seconds>40</start.bundles.timeout.seconds>
                                 <bundle.install.timeout.seconds>30</bundle.install.timeout.seconds>
 
-                                <!-- 
+                                <!--
                                     Define additional bundles to install by specifying the beginning of their artifact name.
                                     The bundles are installed in lexical order of these property names.
-                                    All bundles must be listed as dependencies in this pom, or they won't be installed. 
+                                    All bundles must be listed as dependencies in this pom, or they won't be installed.
                                 -->
                                 <sling.additional.bundle.1>org.apache.sling.junit.core</sling.additional.bundle.1>
                                 <sling.additional.bundle.2>org.apache.sling.junit.scriptable</sling.additional.bundle.2>
-                                <sling.additional.bundle.3>${rootArtifactId}.it.tests</sling.additional.bundle.3>
-                                <sling.additional.bundle.4>${rootArtifactId}.core</sling.additional.bundle.4>
+                                <sling.additional.bundle.3>test.mytest.it.tests</sling.additional.bundle.3>
+                                <sling.additional.bundle.4>test.mytest.core</sling.additional.bundle.4>
```

Comment:
Is this change correct?  I'm thinking it might have crept in by accident.  I'm presuming using the variable would be preferred.

Reply:
My bad, thanks for catching it!